src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -903,9 +914,38 @@ private static bool HasImplicitNullableConversion(TypeSymbol source, TypeSymbol
                 return true;
             }
 
+            if (HasImplicitTupleConversion(unwrappedSource, unwrappedDestination, ref useSiteDiagnostics))
+            {
+                return true;
+            }
+
             return false;
         }
 
+        private bool HasImplicitTupleConversion(TypeSymbol source, TypeSymbol destination, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+        {
+            ImmutableArray<TypeSymbol> sourceTypes;
+            ImmutableArray<TypeSymbol> destTypes;
+
+            if (!source.TryGetElementTypesIfTupleOrCompatible(out sourceTypes) ||
+                !destination.TryGetElementTypesIfTupleOrCompatible(out destTypes) ||
+                sourceTypes.Length != destTypes.Length)
+            {
+                return false;
+            }
+
+            for (int i = 0; i < sourceTypes.Length; i++)
+            {
+                var conversion = ClassifyImplicitConversion(sourceTypes[i], destTypes[i], ref useSiteDiagnostics);
```

Comment:
I'm not sure it is correct to drop the conversion on the floor here (in the sense that it is not clear the underlying conversions can be reconstructed later). I suggest please open an issue to track the suggestion to preserve the underlying conversions in the tuple conversion.